1a.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <vector>
+
+int main()
+{
+	std::cout << "Введите размер вектора\n";
+	int vecsize;
+	std::cin >> vecsize;
+	if (std::cin.fail())
+	{
+		std::cout << "Введено нецелочисленное значение.\n";
+		return 0;
+	}
+	else if (vecsize <= 0)
+	{
+		do
+		{		
+			std::cout << "Размер вектора должен быть положителен. Введите значение заново.\n";
+			std::cin >> vecsize;
+		}
+		while (vecsize <= 0);
+	}
+	std::vector<int> v1;
+	std::cout << "Коллекция чисел: \n";
+	for(unsigned int i = 0; i < vecsize; i++)
+	{
+		int j = i;
+		int value = (vecsize - i) * 2;
+		v1.push_back(value);
+		std::cout << ++j << "е число " << v1.back() << "\n";
+	}
+	std::cout << "Размер вектора: " << v1.size() << "\n";
+
+
+	for (int j = 0; j < vecsize; j++)
+	{
+		int posit = j;
+		for (int k = j+1; k < vecsize; k++)
```

Comment:
Signed-unsigned comparison

Reply:
vecsize is integer. 

![src1qa](https://user-images.githubusercontent.com/25636404/30023058-a32461f4-9176-11e7-90e2-3dbe8ad68638.jpg)